### PR TITLE
MFANA 2nd lecture

### PR DESCRIPTION
**Pull request by LostInDarkMath**

### CODE REVIEW

File: MFANA/chapter1.tex [hunk 1]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+\limn\underbrace{(u_n,c)}_{=\epi(f)}\mit M\times\R.
+\end{align*}
+Da $\epi(f)$ abgeschlossen ist, ist $(u,c)\in\epi(f)$, d.h. $f(u)\leq c$ d.h. $u\in\lbrace f\leq c\rbrace$\\
+
+\underline{Zeige (ii) $\Rightarrow$ (i):}\\
+Sei $(u_n)_{n\in\N}\subseteq\lbrace x\in M:f(x)\leq c\rbrace$ konvergente Folge mit $u:=\lim u_n$ in $M$. Setze
+\begin{align*}
+c:=\liminf\limits_{n\to\infty} f(u_n)\in\R\cup\lbrace\pm\infty\rbrace.
+\end{align*}
+Falls $c>-\infty$, dann enthält $\lbrace f\leq c+\varepsilon\rbrace$ für $\varepsilon>0$ unendlich viele $u_n$. Weil $\lbrace f\leq c+\varepsilon\rbrace$ abgeschlossen ist, ist
```
Comment (lugggas) [inline MFANA/chapter1.tex]: Ok in the lecture we assumed wlog that c < +\infty. trivial case

File: MFANA/chapter1.tex [hunk 2]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
-Sei $M$ ein metrischer Raum, $f\colon M\to\R\cup\lbrace+\infty\rbrace$ eine Funktion. Dann sind äquivalent: \begin{enumerate}
+Sei $M$ ein metrischer Raum, $f\colon M\to\R\cup\lbrace+\infty\rbrace$ eine Funktion. Dann sind äquivalent: 
+\begin{enumerate}[label=(\roman*)]
 	\item $f$ ist unterhalbstetig.
 	\item $\forall c\in\R\colon\lbrace f\leq c\rbrace:=\lbrace u\in M\mid f(u)\leq c\rbrace$ ist abgeschlossen.
 	\item $\lbrace(u,\lambda)\in M\times\R\mid f(u)\leq\lambda\rbrace=:\operatorname{epi}(f)$ ist abgeschlossen.
 \end{enumerate}
 \end{lemma}
+
+\begin{proof}
```
Comment (lugggas) [inline MFANA/chapter1.tex]: The proof for (i) => (iii) is missing

File: MFANA/chapter1.tex [hunk 3]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+\begin{enumerate}[label=(\alph*)]
+\item Für alle $x\in\R\mit f(x)<\infty$ und für alle $y\in\R$ ist
+\begin{align*}
+(0,\infty)\to\R\cup\lbrace +\infty\rbrace,\qquad
+\lambda\mapsto\frac{f(x+\lambda\cdot y)-f(x)}{\lambda}
+\end{align*}
+monoton wachsend.
+\item $\forall x\in\R\mit f(x)<\infty$ existieren die Grenzwerte
+\begin{align*}
+\lim\limits_{\lambda\to0^+}\frac{f(x+\lambda\cdot y)-f(x)}{\lambda}\in\R\cup\lbrace\pm\infty\rbrace
```
Comment (lugggas) [inline MFANA/chapter1.tex]: No y here.

File: MFANA/chapter1.tex [hunk 4]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+\text{ und }
+\lim\limits_{\lambda\to0^+}\frac{f(x-\lambda\cdot y)-f(x)}{-\lambda}\in\R\cup\lbrace\pm\infty\rbrace.
```
Comment (lugggas) [inline MFANA/chapter1.tex]: no y here either

File: MFANA/chapter1.tex [hunk 5]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+\end{align*}
+\item 
+$\begin{aligned}
+\dom(f):=\lbrace r\in\R:f(r)<\infty\rbrace
+\end{aligned}$ 
+ist ein Intervall und $f$ ist stetig auf $\dom(f)$.
+\end{enumerate}
+\end{lemma}
+\begin{proof}
+\underline{Zeige (a):} O.B.d.A. sei $x=0,~f(x)=0$ und $y=1$.\\
```
Comment (lugggas) [inline MFANA/chapter1.tex]: perhaps one can give a little explanation

File: MFANA/chapter1.tex [hunk 6]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+Zu zeigen ist, dass $\lambda\mapsto\frac{f(\lambda)}{\lambda}$ monoton wachsend auf $(0,\infty)$ ist.\\
+Sei $0<\lambda_1<\lambda_2$. Dann ist 
+\begin{align*}
+\lambda_1=(1-\lambda)\cdot 0+\lambda\cdot\lambda_2\text{ für }\lambda:=\frac{\lambda_1}{\lambda_2}\in[0,1]
+\end{align*}
+und somit
+\begin{align*}
+f(x_1)
```
Comment (lugggas) [inline MFANA/chapter1.tex]: f(\lambda_1)

File: MFANA/chapter1.tex [hunk 7]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+\end{align*}
+Behauptung (c) folgt dann aus (b), welche aus (a) folgt.
+\end{proof}
+
+\begin{proof}[Beweis des Lemmas über die Eigenschaften des Brackets]\enter
+\underline{Zeige (a):} Das Bracket ist oberhalbstetig, denn
+\begin{align*}
+[\cdot,\cdot]_\lambda:X\times X\to\R,~(u,v)\mapsto[u,v]_\lambda:=\frac{\Vert u+\lambda\cdot v\Vert - \Vert u\Vert}{\lambda}
+\end{align*}
+ist stetig (da jede Norm stetig ist) und damit auch oberhalbstetig für alle $\lambda>0$. Das Infimum von oberhalbstetiger Funktion ist wieder oberhalbstetig.\\
```
Comment (lugggas) [inline MFANA/chapter1.tex]: Nice!

File: MFANA/chapter1.tex [hunk 8]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+\inf\limits_{\lambda>0}[u,\mu\cdot v]_\lambda\\
+&=\inf\limits_{\lambda>0}\frac{\Vert u+\lambda\cdot\mu\cdot v\Vert-\Vert u\Vert}{\lambda}\cdot\frac{\mu}{\mu}\\
+&=\inf\limits_{\lambda>0}\mu\cdot[u,v]_{\lambda\cdot\mu}\\
+&=\mu\cdot[u,v]
+\end{align*}
+und für alle $\mu\in(0,1)$ gilt
+\begin{align*}
+[u,v_1+v_2]
+&=
+\inf\limits_{\lambda>0}\frac{\Vert u+\lambda\cdot(v_1+v_2)\Vert-\Vert u\Vert}{\lambda}\cdot\frac{\mu}{\mu}\\
```
Comment (lugggas) [inline MFANA/chapter1.tex]: I dont like the mu/mu stuff. What is it supposed to say?

File: MFANA/chapter1.tex [hunk 9]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+&\stackrel{\mu\in(0,1)}{\leq}
+\underbrace{\inf\limits_{\lambda>0}}_{=\lim\limits_{\lambda\to0}}\Vert\mu\cdot u+\lambda\cdot v_1\Vert+\Vert(1-\mu)\cdot u+\lambda\cdot v_2\Vert-\mu\cdot\Vert u\Vert-(1-\mu)\cdot\Vert u\Vert\\
```
Comment (lugggas) [inline MFANA/chapter1.tex]: here everthing has to be divided by \lambda

File: MFANA/chapter1.tex [hunk 10]
```diff
@@ -57,9 +58,217 @@ \section{Das ``Bracket''}
...
+=\Vert u\Vert
+\end{align*}
+\end{proof}
+
+\begin{bemerkung}
+Falls $X=H$ ein Hilbertraum mit Skalarprodukt $\langle\cdot,\cdot\rangle$ ist, dann ist
+\begin{align*}
+[u,v] 
+&=
+\lim\limits_{\lambda\to0^+}\sqrt{\langle u+\lambda\cdot v,u+\lambda\cdot v\rangle}-\sqrt{\langle u,u\rangle}\\
```
Comment (lugggas) [inline MFANA/chapter1.tex]: devied by \lambda is missing

File: MFANA/chapter1.tex (hunk 8 above)
Comment (LostInDarkMath) [reply MFANA/chapter1.tex]: It's just a "gehaltvolle Eins" to apply the definition of the bracket. Nothing special. And i like this notation for this.

Comment (LostInDarkMath) [reply MFANA/chapter1.tex]: Sry, i looked at the wring line. You're right.

**Comment by LostInDarkMath** [comment]
I fixed them all (hopefully).

Thank you @lugggas for your great feedback :)

**Pull request merged**